tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -160,6 +160,10 @@ class TrtConversionParams(
       inputs during runtime, then a new TensorRT engine is built at runtime if
       allow_build_at_runtime=True, and otherwise native TF is used. This
       argument is only effective if is_dynamic_op=True.
+    allow_mixed_precision_on_native_ops: whether to allow TensorFlow to
+      activate AMP (Automatic Mixed Precision) on the operations which are not
```

Comment:
How about this:
When the precison_mode is FP16 or INT8, whether to allow tensorflow to implement FP32 operations that are not converted to TensorRT with low precisions.